sdcm/sct_events.py

```diff
@@ -235,6 +235,30 @@ def __str__(self):
                 super(DisruptionEvent, self).__str__(), self)
 
 
+class ClusterHealthValidatorEvent(SctEvent):
+    def __init__(self, type, name, status, node=None, message=None, error=None, **kwargs):
+        super(ClusterHealthValidatorEvent, self).__init__()
+        self.name = name
+        self.type = type
+        self.node = str(node)
+        self.severity = Severity.NORMAL if status else Severity.CRITICAL
+        if error:
```

Comment:
dynamic class attribute????

Reply:
What do you mean? It's same as in DisruptionEvent. Do fix  it also there?

Reply:
If somebody did a mistake, that doesn't mean that you should copy it :)

Reply:
please fix

Reply:
Fixed